### ISSUE TITLE
Search / `find` utility

### ISSUE THREAD

**Issue by leezu** [juruen/rmapi#47]
It would be nice to have support for the unix `find` utility or some other, possibly fuzzy search by document names. (This would be addressed "automatically" once the fuse support is ready).

Thanks for your work on rmapi!

**Comment by juruen** [comment]
@leezu, just fixed those two.

I also updated the doc to explain how to make the regexp case insensitive: `(?i)foo`.

**Comment by leezu** [comment]
Thanks @juruen
